.editorconfig

```diff
@@ -1,11 +1,12 @@
 root = true
 
 [*]
-charset = utf-8
 indent_style = space
 indent_size = 2
-insert_final_newline = true
+end_of_line = lf
+charset = utf-8
 trim_trailing_whitespace = true
+insert_final_newline = true
 
-[*.js]
-indent_size = 4
+[*.md]
+trim_trailing_whitespace = false
```

Comment:
Might be usefull if you want to use for instance [Line Return](http://markdown-guide.readthedocs.io/en/latest/basics.html#line-return).